### PR TITLE
added support for context injection on the identityClient

### PR DESCRIPTION
**Pull request by aviadmizrachi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,4 +140,36 @@ await httpClient.post('identity/resources/auth/v1/user', {
...
+
+First, let's import the `IdentityClient`
+```javascript
+const { IdentityClient } = require('@frontegg/client');
+```
+
+Then, initialize the client
+```javascript
+const identityClient = new IdentityClient({ FRONTEGG_CLIENT_ID: 'your-client-id', FRONTEGG_API_KEY: 'your-api-key' });
+identityClient.validateIdentityOnToken(token, { roles: ['admin'], permissions: ['read'] });
```
Comment (amirjaron) [inline README.md:156]: maybe remove this line here, the example is later

**Review by amirjaron — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (eran-frontegg) [inline README.md:156]: why do we need this line? It is used later in line 165 (user = IdentityClient.validateIdentity....)

Comment (amirjaron) [reply README.md:156]: haha I just wrote that

**Review by eran-frontegg — CHANGES_REQUESTED** [review]
please update the commit message to fit conventional commit style - feat(clients): add support for context injection on the identity client (semantic release won't pick up the changes and won't release)

**Review by eran-frontegg — COMMENTED** [review]
please update the commit message before merging with squash, to fit conventional commit style - feat(clients): add support for context injection on the identity client (semantic release won't pick up the changes and won't release)

**Review by eran-frontegg — APPROVED** [review]

**Pull request merged**